### ISSUE TITLE
Random mouse angles on player movement is not a bug

### ISSUE THREAD

**Issue by TheK0tYaRa** [cabaletta/baritone#1493]
I suggest you ignore this unfunny way that Microsoft chose to remove all joy of playing minecraft for the first of April.

**Comment by S-B99** [comment]
Oh wait did minecraft do that? LMAOOO

**Comment by TheK0tYaRa** [comment]
Yep. And the most strange part is that the problem vanishes if the system is under high load. Like an OBS running on the CPU.

**Comment by TheK0tYaRa** [comment]
Currently that's not an issue because 01.04.20 has passed

**Comment by TheK0tYaRa** [comment]
Wait a second, it seems that this issue persists. wtf.

**Comment by S-B99** [comment]
bru
